### PR TITLE
Capital Module Rework/Minor edits to Sub-cap Modules

### PR DESCRIPTION
**Pull request by taelon875**
Fixes:
Bulwark and Repair Field now don't loop to provide extra healing, They also fixed Visual Timers.

Changed:
All Capital Ship Modules lie in their own category in the Engineering Bench.

Targeting Array - No longer gives Attack Boost (see Weapons Computer) instead of evasion decrease it is now damage.

Advanced Thrusters -  No Value Changes here, just a description edit.

Reinforced Plating -  No longer gives evasion decrease, but does decrease shields instead.

Quad Lasers:  Adjusted oversized Fire Range and increase capacitor costs.

Turbolasers: Increased Damage from Module Bonus.

Laser Cannon Battery: Adjusted Range and Damage. (Slight increase on Lower-End Boss Damages.)

Beam Cannon/Laser Cannon: Adjusted Ranges and Cap Costs.

Bulwark/Repair Field - Both now heal larger amounts to cover for single use fire mechanics opposed to original design.

Missile Weapons - Adjusted the "minimum shield value" to 10.

Capital Ship Configuration Changes -
Scaling for Hull/Shields Increased Overall.
Warship - No Longer gains Attack Bonus.  Slight increase in Armor values and health boost. But Gains Accuracy Increase. Logistics - Small Shield Increase and Armor Increase. Skirmisher -  Lost Accuracy Increase, Gained Attack Increase and larger shield Pool.

Added:
Weapons Computer - Increased Damage for Lower Accuracy Power Diverter  - Lower Cap for Faster shield recharge E-War Module - Generates Enmity

https://docs.google.com/spreadsheets/d/1g0DB4i-zY4bQQzifilb1OrbiFn28FHK5Zg_29RxK4Yo/edit?usp=sharing

### CODE REVIEW

File: SWLOR.Game.Server/Feature/ShipModuleDefinition/WeaponsComputerModuleDefinition.cs [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        private void WeaponsComputer(string itemTag,
+            string name,
+            string shortName,
+            int boostAmount)
+        {
+            _builder.Create(itemTag)
+                .Name(name)
+                .ShortName(shortName)
+                .Texture("iit_ess_252")
+                .Description("This weapons computer increases all Attack by 20 but reduce Accuracy by 10.")
```
Comment (zunath) [inline SWLOR.Game.Server/Feature/ShipModuleDefinition/WeaponsComputerModuleDefinition.cs:27]: reduce -> reduces

File: SWLOR.Game.Server/Service/CraftService/RecipeCategoryType.cs [hunk 2]
```diff
@@ -130,6 +130,8 @@ public enum RecipeCategoryType
         SpecialSubmissionItems = 60,
         [RecipeCategory("Starship Ammo", true)]
         StarshipAmmo = 61,
+        [RecipeCategory("Capital Module Recipes", true)]
+        CapitalShipModule =62,
```
Comment (zunath) [inline SWLOR.Game.Server/Service/CraftService/RecipeCategoryType.cs:134]: space between the equals and the 62

File: SWLOR.Game.Server/Service/CraftService/RecipeType.cs [hunk 3]
```diff
@@ -2502,6 +2502,11 @@ public enum RecipeType
         CorvetteJehaveyFrigate = 4794,
```
Comment (zunath) [inline SWLOR.Game.Server/Service/CraftService/RecipeType.cs:2503]: you don't need a gap of empty space here

**Review by zunath — DISMISSED** [review]
Requesting changes

File: SWLOR.Game.Server/Feature/MigrationDefinition/ServerMigration/_17_StatsUpdateCapitals.cs [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+namespace SWLOR.Game.Server.Feature.MigrationDefinition.ServerMigration
+{
+    public class _17_StatsUpdateCapitals : ServerMigrationBase, IServerMigration
+    {
+        public int Version => 17;
+        public MigrationExecutionType ExecutionType => MigrationExecutionType.PostCacheLoad;
+        public void Migrate()
+        {
+            RecalculateAllShipStats();
```
[the remark this answers is not in the document]
Comment (taelon875) [reply SWLOR.Game.Server/Feature/MigrationDefinition/ServerMigration/_17_StatsUpdateCapitals.cs:13]: The issue that happened before was that it didn't account for the configuration modules, we added that in and it fixed it.

**Pull request merged**